packages/web-components/src/components/tabs-extended/tabs-extended.ts

```diff
@@ -73,11 +78,9 @@ class DDSTabsExtended extends MediaQueryMixin(StableSelectorMixin(LitElement), {
     this._tabItems = slottedNodes.filter(
       (node) => node instanceof DDSTab
     ) as DDSTab[];
-    this._tabItems.forEach((tab, index) => {
-      this._activeTabIndex = (tab as DDSTab).selected
-        ? index
-        : this._activeTabIndex;
-    });
+    this._activeTabIndex =
+      this._tabItems.findIndex((tab) => (tab as DDSTab).selected) ??
+      this._activeTabIndex;
```

Comment:
Seems like since [`findIndex` returns either the index, or -1](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex#return_value), the right hand side of the [nullish coalescing operator ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) would never get evaluated. Can this be simplified to the following?
```suggestion
      this._activeTabIndex = this._tabItems.findIndex((tab) => (tab as DDSTab).selected);
```

Reply:
Good catch and good call!